feature/google/src/testLatestDependencies/java/BillingWrapperTest.kt

```diff
@@ -0,0 +1,1926 @@
+import android.app.Activity
```

Comment:
This file is the same as the current BC4 one right?

Reply:
yep!